File: responses/YTCUQF-XM4/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    ans = 0
    for i in range(x):
        if i % 3 == 0:
            ans += i
        elif i % 5 == 0:
            ans += i
    return ans

File: responses/HXBHHF-BFH/reverse_digits.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(x)
  numbers = []
  1.upto(x) do |i|
    if (i%3).zero? ^ (i%5).zero?
      numbers.push i
    end
  end
  numbers.reduce(:+)
end